app/models/csv_export.rb

```diff
@@ -0,0 +1,39 @@
+class CsvExport < ActiveRecord::Base
+  belongs_to :user
+  STATUS_VALUES = ['pending', 'started', 'finished', 'downloaded', 'failed', 'deleted']
```

Comment:
or you can use [activerecord enum](http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)